jdaviz/configs/imviz/plugins/reproject/reproject.py

```diff
@@ -0,0 +1,94 @@
+import numpy as np
+import time
+from glue.core.data import Data
+from traitlets import Bool
+
+from jdaviz.core.events import SnackbarMessage
+from jdaviz.core.registries import tray_registry
+from jdaviz.core.template_mixin import (PluginTemplateMixin, DatasetSelectMixin,
+                                        AutoTextFieldMixin)
+
+try:
+    from reproject import reproject_interp
+    from reproject.mosaicking import find_optimal_celestial_wcs
+except ImportError:
+    HAS_REPROJECT = False
+else:
+    HAS_REPROJECT = True
+
+__all__ = ['Reproject']
+
+
+@tray_registry('imviz-reproject', label="Reproject")
+class Reproject(PluginTemplateMixin, DatasetSelectMixin, AutoTextFieldMixin):
```

Comment:
I am guessing just throwing `AutoTextFieldMixin` in there isn't enough? I am getting error about not enough arguments for the constructor. And I can't quite figure it out from the API doc. Maybe I am using this wrong, @kecnry ?

Reply:
Looking into it.